ci/dependency-update/Jenkinsfile

```diff
@@ -159,6 +163,21 @@ pipeline {
 							}
 						}
 					}
+                    stage('Build Hibernate ORM') {
+                        when {
+                            expression {
+                                return settings().requiresLocalOrmBuild
+                            }
+                        }
+                        steps {
+                            script {
+                                sh "git clone ${params.ORM_REPOSITORY} --branch ${params.ORM_BRANCH} --single-branch ${env.WORKSPACE_TMP}/hibernate-orm-local-copy"
+                            }
+                            script {
+                                sh ".${env.WORKSPACE_TMP}/hibernate-orm-local-copy/gradlew publishToMavenLocal"
```

Comment:
that's the command I usually use for adding orm jars to local maven repo, and it doesn't execute tests. I'll add the `-x test` just in case, since I'm updating the file anyway 😃